src/objects/Points.js

```diff
@@ -79,17 +81,17 @@ Points.prototype = Object.assign( Object.create( Object3D.prototype ), {
 
 					const a = indices[ i ];
 
-					_position.fromArray( positions, a * 3 );
+					_position.fromArray( positions, a * stride + offset );
```

Comment:
It's atually better to use `Vector3.fromBufferAttribute()` in order to hide the interleaved logic.

Reply:
I ran some informal performance tests based on some feedback from @usefulthink ..
On a 5 million random point cloud:
fromArray with inline address calculation took min 65 to max 81   msec
fromBufferAttribute with getX/Y/Z            took min 79 to max 110 msec
fromBufferAttribute optimized with a .toVector3 method on BufferAttribute and InterleavedBufferAttribute for faster conversion to Vector3                                took min 70 to max 93 msec.

Any opinions/guidance on what approach to take @Mugen87 ?

Reply:
Option 3 is in between both methods in terms of speed, but requires adding a .toVector3 method to BufferGeometry and InterleavedBufferGeometry...

Reply:
Sorry, but performance is not the most important point here. It's code readability and consistency.  Hence, please use `Vector3.fromBufferAttribute()`.

Reply:
Indeed. Unless it was 10x slower or something... These numbers seem like it's readability should win here.

Reply:
Cool. Thanks ya'll! :)